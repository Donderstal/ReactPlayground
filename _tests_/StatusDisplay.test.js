import React from 'react';
import renderer from 'react-test-renderer';


import StatusDisplay from '../src/components/StatusDisplay';


test('test if sDisplay renders', () => {
     const tree = renderer.create(<StatusDisplay />).toJSON();
      expect(tree).toMatchSnapshot();
});