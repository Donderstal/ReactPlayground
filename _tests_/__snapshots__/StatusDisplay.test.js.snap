// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test if sDisplay renders 1`] = `
<View
  style={
    Object {
      "backgroundColor": "lightblue",
      "borderRightColor": "#f4f984",
      "borderRightWidth": 0.5,
      "flex": 0.25,
      "margin": "5%",
      "padding": "5%",
    }
  }
>
  <Text>
    No process to display...
  </Text>
</View>
`;
